src/unix/apple/cpu.rs

```diff
@@ -371,6 +382,29 @@ pub(crate) fn get_vendor_id_and_brand() -> (String, String) {
     (vendor, get_sysctl_str(b"machdep.cpu.brand_string\0"))
 }
 
+pub(crate) unsafe fn get_cpu_arch() -> CpuArch {
+    use std::ffi::CStr;
+    let mut mib: [c_int; 2] = [libc::CTL_HW, libc::HW_MACHINE_ARCH];
```

Comment:
This is not the same constant name (`HW_MACHINE_ARCH`) as below (`HW_MACHINE`). Is it normal?

Reply:
According to what I see on internet search, `HW_MACHINE_ARCH` can be used to get the system arch on Mac and FreeBSD